test/test_prototype_transforms_functional.py

```diff
@@ -155,14 +156,12 @@ def _unbatch(self, batch, *, data_dims):
         if batched_tensor.ndim == data_dims:
             return batch
 
-        unbatcheds = []
```

Comment:
Again, just undoing what #7227 did, since we no longer care for datapoints at this stage.